tests/unit/beacons/test_swapusage.py

```diff
@@ -0,0 +1,66 @@
+# Python libs
```

Comment:
Since this is a new test module would you mind migrating it to pytest? `tests/pytests/unit/beacons`

Reply:
@Ch3LL Done.  Since this was based on `memusage`, I've migrated that to pytest as well.  If there are any changes that need to be made, I'll apply them for both test files.

Reply:
thanks!!!!